charts/unified-chart/templates/ingress/ingress.yaml

```diff
@@ -30,12 +30,14 @@ spec:
     - host: {{ $ingress.hostnameOverride | quote }}
       http:
         paths:
-        - path: /
-          backend:
-            service:
-              name: frontegg-{{ include "name" $top }}
-              port:
-                number: {{ $values.service.port }}
-          pathType: ImplementationSpecific
+        {{- range $path := $ingress.paths }}
+          - path: {{ .path }}
+            pathType: {{ .pathType }}
+            backend:
+              service:
+                name: {{ .service.name }}
```

Comment:
this wont work. 
range changes ur scope
it should be $path.path
$path.pathtype

Reply:
actually it might work because in this case $path and . are the same scope
i think better use the iterator variable